### DISCUSSION TITLE
Connect failed in Android devices while iOS works as normal

### DISCUSSION THREAD

**Discussion by wongzigii** [WalletConnect/walletconnect-monorepo#3836] [Developer Support] [answered]
We have encountered an issue that Android devices failed to connect while the iOS devices works as normal, both with the same ProjectId.

Some similar issues: 

Android: 

https://github.com/WalletConnect/WalletConnectKotlinV2/issues/570
https://github.com/WalletConnect/WalletConnectKotlinV2/issues/976

iOS: 

https://github.com/WalletConnect/WalletConnectSwiftV2/issues/1062

**Comment by ckorhonen** [comment]
We are also experiencing this in our app. The ProjectId is working on Web and iOS, but fails on Android.

**Comment by brachsterX** [comment]
Libraries / Frameworks used:

WalletConnectKotlinV2

com.walletconnect:android-bom:1.12.0

**Comment by ablankstory** [comment]
@glitch-txs - this is the issue @brachsterX and I are asking about in Discord. 

Thanks for taking a look! 🤝

**Comment by glitch-txs** [comment]
@ablankstory Could you share more information on this issue? have you tried using our wallet examples? what error are you getting? could you share the steps to reproduce?

**Reply by ablankstory** [reply]
@glitch-txs thanks for adding on here. @wongzigii is one of our app developers so he'll have more info. I also shared this link/issue with another one of the devs too that works on Android.

We'll get you more info soon!

**Comment by Yonghengl** [comment]
At Android:
I user demo (https://github.com/WalletConnect/WalletConnectKotlinV2/tree/develop/sample/wallet) connect to https://react-app.walletconnect.com/
When I use ProjectId： e81dbe11180fe8ab227e83f9f3f36c6b  can connect.
But ProjectId :80b809d5dbb364658b31cc59b7d9c143 cannot connect. Error :Expected HTTP 101 response but was ‘403 Forbidden

**Comment by Yonghengl** [comment]
<img width="371" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/55374358/24a300be-1d8e-466f-9727-b6ab1c913662">

**Comment by ablankstory** [comment]
@glitch-txs - please see above, a response from the team. So our short-term fix was to use a new project id. But it would be great if this can get fully debugged so it doesn't happen again.

Thanks again for the assistance here.

**Comment by glitch-txs** [comment]
Hi, is this still an issue?

**Comment by ablankstory** [comment]
@glitch-txs since we created a new Project ID, Android users have been able to connect. But I'm not sure if the issue will come back.

Thanks.

**Comment by Yonghengl** [comment, marked as answer]
Now this projectId can be connected successfully

**Comment by spectrl** [comment]
> @glitch-txs since we created a new Project ID, Android users have been able to connect. But I'm not sure if the issue will come back.
> 
> Thanks.

@glitch-txs We did the same but the issue has now come back even on our new project id
